docs/system_requirements/using_podman.md

```diff
@@ -36,4 +35,47 @@ func TestSomething(t *testing.T) {
 }
 ```
 
-The `ProviderPodman` configures the `DockerProvider` with the correct default network for Podman to ensure also complex network scenarios are working as with Docker.
\ No newline at end of file
+The `ProviderPodman` configures the `DockerProvider` with the correct default network for Podman to ensure complex network scenarios are working as with Docker.
+
+## Podman socket activation
+
+The reaper container needs to connect to the docker daemon to reape containers, so the podman socket service must be started:
```

Comment:
```suggestion
The reaper container needs to connect to the docker daemon to reaper containers, so the podman socket service must be started:
```

Reply:
It looks like I used an obsolete spelling of reap. But reaper is not correct in this context. I will fix it.